build.gradle

```diff
@@ -25,6 +25,7 @@ allprojects {
         google()
         jcenter()
         maven { url "https://dl.bintray.com/wordpress-mobile/maven" }
+        maven { url "https://jitpack.io" }
```

Comment:
Should `maven { url "https://jitpack.io" }` now be removed from `WordPress/build.gradle` so that it is not duplicated?

Reply:
Most probably, yeah. Let's resolve [the other comment](https://github.com/wordpress-mobile/WordPress-Android/pull/8762#pullrequestreview-183208401) first since where the JitPack maven repo definition will land depends on whether we will have CDN repos in the top level Gradle script or not, cool?

Reply:
Done with 8668940c7e.